src/client/components/Home/RegisterButton.style.css

```diff
@@ -0,0 +1,18 @@
+.homepage-slogan{
+  height: 63vh;
+  width: 100vw;
+  text-align: center;
+  align-items: center;
+  color: white;
+  padding-top: 110px;
+  z-index: 2;
+}
+
+.homepage-slogan:nth-of-type(1){
+  background-color:  #6236FF;
+  background-image: url("../../assets/images/portraits/homeimage.png");
```

Comment:
Would be nice if image cover all width.

Reply:
the original image size is very big.if i center the position it's not even displaying the laptop in the picture.so, i cropped the actual image and managed to get this picture.